chainntnfs/test_utils.go

```diff
@@ -197,7 +198,8 @@ func NewBitcoindBackend(t *testing.T, minerAddr string, txindex,
 
 	t.Helper()
 
-	tempBitcoindDir := t.TempDir()
+	tempBitcoindDir, err := ioutil.TempDir("", "bitcoind")
```

Comment:
Could we add some notes here to say why we don't use `t.TempDir` just in case someone else changes it again in the future😂

Reply:
great idea 😂